Userland/Libraries/LibCore/Account.cpp

```diff
@@ -152,7 +152,7 @@ bool Account::login() const
         return false;
 
     auto const temporary_directory = String::formatted("/tmp/{}", m_uid);
-    if (auto result = Core::Directory::create(temporary_directory, Core::Directory::CreateDirectories::No); result.is_error())
+    if (auto result = Core::Directory::create(temporary_directory, Core::Directory::CreateDirectories::Yes); result.is_error())
         dbgln("{}", result.release_error());
```

Comment:
Hold up, what if another malicious user has already created this directory?

Reply:
You're right, we need something more secure.

Maybe we can create a directory at first login and store the value in a static map in `Account`. It's default value would be the current value (just the UID) and if it's taken we iterate on UID-X until we find something. Then further call of `parse_uid_in_path` will restore the value from the map.

Reply:
How about we create the directory somewhere in the user's home directory instead? Is there some reason we have to use the shared playground in `/tmp`?

Reply:
The only issue I can think of (it also exists when using /tmp) is stale sockets living after the crash of the owner.

Using a path outside /tmp, prevent a magic reboot to solve the issue.

I'm not very familiar with Unix sockets and even less with serenity's one.  I could definitely have missed something.

Reply:
From what I read [here](https://stackoverflow.com/questions/42555403/unix-domain-socket-file-not-removed-on-application-crash), I suppose that it would work.

Abstract socket might also be worth implementing.

Anyway, I will move it to the user's personal directory and check that leftover portals are indeed cleaned before creation.

Reply:
I finally moved this commit to #14673.